diagnostics/app/model/diagnostics/DynamoDbReport.scala

```diff
@@ -72,7 +72,7 @@ object CSPDynamoDbReport extends DynamoDbReport[CSPReport] {
       .withAttributeUpdates(count.asJava)
 
     client.updateItemFuture(update) onFailure {
-      case error: Throwable => log.error(s"Failed to log CSP report to DynamoDB", error)
+      case error: Throwable => log.error(s"Failed to log CSP report to DynamoDB: $error", error)
```

Comment:
because it's being interpolated it'll call `toString` on the error.
the second arg is meant to show the full trace in cloudwatch, but that's not working